lti/sentry.py

```diff
@@ -0,0 +1,19 @@
+"""Report exceptions to Sentry."""
+
+import os
+
+import raven
+
+
+def get_raven_client(request):
+    """Return the Raven client for reporting crashes to Sentry."""
+    client = request.registry["raven.client"]
+    client.http_context({"url": request.url, "method": request.method})
```

Comment:
Might be good to include a bit more information, similar to https://github.com/hypothesis/h/blob/master/h/sentry.py#L24-L32